extension/packages/hardhat/contracts/RiggedRoll.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity >=0.8.0 <0.9.0; //Do not change the solidity version as it negativly impacts submission grading
+//SPDX-License-Identifier: MIT
+
+import "hardhat/console.sol";
+import "./DiceGame.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract RiggedRoll is Ownable {
+    DiceGame public diceGame;
+
+    constructor(address payable diceGameAddress) Ownable(diceGameAddress) {
```

Comment:
it should be `Ownable(msg.sender)` , right?

Reply:
Yes! Thanks!!

Reply:
Fixed!